packages/kbn-logging/src/logger.ts

```diff
@@ -7,15 +7,14 @@
  */
 
 import { LogRecord } from './log_record';
+import { Ecs } from './ecs';
 
 /**
  * Contextual metadata
  *
  * @public
  */
-export interface LogMeta {
-  [key: string]: any;
-}
+export type LogMeta = Partial<Omit<Ecs, 'ecs'>>;
```

Comment:
> Let's a public interface with ecs field internally

The only issue I can see with this is a maintenance one -- if we update versions of ECS (or if someday the types are auto generated), then we will need to manually update the top-level keys in `LogMeta` as well. However, since top-level keys don't change all that much, it probably wouldn't hurt to duplicate them for now, and reconsider later if it turns out to be a problem in practice. Will update.